student.rb

```diff
@@ -18,8 +21,9 @@ def play_hooky
     puts '¯\(ツ)/¯'
   end
 
-  def classroom=(classroom)
-    @classroom = classroom
-    classroom.students.push(self) unless classroom.students.include?(self)
-  end
+  include Assignment
+  # def classroom=(classroom)
+  #   @classroom = classroom
+  #   classroom.students.push(self) unless classroom.students.include?(self)
+  # end
```

Comment:
- [ ] Please kindly remove this commented code to make your codebase clean. 👍

Reply:
Thankyou very much 👍 @OyePriscilla. We will implement the required changes and request for another review.